Learnings from Unilever SAP Migration (#1110)
MicrosoftDocs/cloud-adoption-framework: docs/scenarios/sap/eslz-management-and-monitoring.md

- Use [Azure Site Recovery](/azure/site-recovery/site-recovery-overview) monitoring to maintain disaster recovery service health for SAP application servers.

- All production systems must use premium managed disk with minimum of 128GB (P10) for OS Disks to avoid any performance issues.

> - Exclude all the database filesystems and executables from anti-virus scan, if not it could lead to performance problems . Please check with the database vendors for a prescriptive details on the exclusion list. For example: Oracle recommends excluding /oracle/<sid>/sapdata* from anti-virus scans.

Reviewer: ```suggestion
- Exclude all the database filesystems and executables from anti-virus scan. Otherwise it could lead to performance problems. Check with the database vendors for a prescriptive details on the exclusion list. For example, Oracle recommends excluding /oracle/\<sid>/sapdata* from anti-virus scans.
```
Adding a backslash to make the bracketed portion render properly. Fixing some punctuation.

- Always ensure time-zone matches between Operating system and SAP system.

- Do not group different application services in the same cluster.

>   - Do not combine DRBD and Central Services clusters on the same cluster,However you can use the same pacemaker cluster to manage  ~5 different central services([multi-sid cluster](https://learn.microsoft.com/en-us/azure/sap/workloads/high-availability-guide-rhel-multi-sid)).

Reviewer: ```suggestion
  - Do not combine DRBD and Central Services clusters on the same cluster. However, you can use the same pacemaker cluster to manage  approximately five different central services ([multi-sid cluster](./workloads/high-availability-guide-rhel-multi-sid.md)).
```
Fixing link, fixing some grammar and punctuation.
